specs/glossary.md

```diff
@@ -72,12 +72,12 @@ block, and output block properties, which are derived after executing the block'
 
 "Externally Owned Account", an Ethereum term to designate addresses operated by users, as opposed to contract addresses.
 
-## Merkle Root
+## Merkle Patricia Trie
 
-[Merkle root]: glossary.md#merkle-roots
+[mpt]: glossary.md#merkle-patricia-trie
 
-The Merkle root is the root hash of a [Merkle Patricia tree] (MPT). A MPT is a sparse [trie], which is a tree-like
-structure that maps keys to values. The root hash of a MPT is a commitment to the contents of the tree, which allows a
+A [Merkle Patricia tree] (MPT) is a sparse trie, which is a tree-like structure that maps keys to values.
```

Comment:
Since the heading is now "Merkle Patricia Trie", do you want to change this link say the same? (don't forget the anchor)